ui/src/components/VaultSummary.jsx

```diff
@@ -68,11 +65,14 @@ const useStyles = makeStyles(theme => {
   };
 });
 
-const calcRatio = (priceRate, newLock, newBorrow) => {
-  const lockPrice = floorMultiplyBy(newLock, priceRate);
-  const newRatio = makeRatioFromAmounts(lockPrice, newBorrow);
-  return newRatio;
-};
+const calcRatio = (priceRate, newLock, newBorrow) =>
```

Comment:
@Chris-Hibbert I think I understand what you're saying, does this seem better? We don't always have an old value to use `quantize`, but I don't think the numbers being too large is going to an issue with BigInts

Reply:
I think this is good.